lib/push/push-to-space.js

```diff
@@ -100,7 +100,7 @@ export default function ({
         const draftsAssets = sourceContent.assets.filter(({original: asset}) =>
         (asset.isDraft && asset.isDraft()))
         const publishedAssets = sourceContent.assets.filter(
-        ({original: asset}) => !(asset.isDraft && asset.isDraft()))
+        ({original: asset}) => asset.isPublished())
```

Comment:
Could we do the same in other places where we do the `isDraft && isDraft()` check?